packages/cheerio-crawler/package.json

```diff
@@ -55,7 +55,7 @@
     "dependencies": {
         "@apify/timeout": "^0.3.0",
```

Comment:
most of those deps are probably no longer directly used, so we dont need to have them in dependencies here